### PR TITLE
fixes #13131 - plugin skips are back

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -85,6 +87,13 @@ def reconsider_gc_deferment
       end
     end
 
+    def skip_if_plugin_asked_to
+      skips = Foreman::Plugin.tests_to_skip[self.class.name].to_a
+      if skips.any?{|name| @NAME.end_with?(name)}
+        skip "Here be dragons"
```
Comment (domcleal) [inline test/test_helper.rb]: This message is inappropriate, it should provide more information.

**Comment by domcleal** [comment]
The warning in the plugin descriptor should be removed too.

File: test/test_helper.rb (hunk 1 above)
Comment (unorthodoxgeek) [inline test/test_helper.rb]: it's not inappropriate, just silly :stuck_out_tongue_closed_eyes: 

anyway... this message isn't actually displayed anywhere, so it's harmless. Every codebase I work with has a "here be dragons" somewhere in it, and it's about time foreman gets one...

**Comment by unorthodoxgeek** [comment]
@domcleal - removed the warning

File: test/test_helper.rb (hunk 1 above)
Comment (domcleal) [inline test/test_helper.rb]: It should be displayed in the rake test output, in Jenkins etc.  It should state the facts, i.e. the test was disabled by plugin X.

Comment (unorthodoxgeek) [inline test/test_helper.rb]: you're no fun :-1:

**Comment by domcleal** [comment]
Merged as a0a3a87, thanks @unorthodoxgeek.

**Pull request closed without merging**